corehq/apps/hqadmin/management/commands/record_deploy_success.py

```diff
@@ -31,18 +31,21 @@ class Command(BaseCommand):
         make_option('--environment', help='Environment {production|staging etc...}', default=settings.SERVER_ENVIRONMENT),
         make_option('--mail_admins', help='Mail Admins', default=False, action='store_true'),
         make_option('--url', help='A link to a URL for the deploy', default=False),
+        make_option('--current_deploy', help='ref of current deploy', default=None),
+        make_option('--previous_deploy', help='ref of previous deploy', default=None),
+        make_option('--pr_numbers', help='List of PRs included in deploy', default=None),
```

Comment:
actually, i think i'll just use unauthenticated GitHub. this happens infrequently enough that we probably won't need to authenticate